Formula/threesdk.rb

```diff
@@ -0,0 +1,140 @@
+class Threesdk < Formula
```

Comment:
Since the repo is called `jumpScaleX_core` Shouldn't the formula be called `JumpScaleCore`? Why the different name?

Reply:
just that it's the commercial name of our software

Reply:
Either rename the repo or rename the formula.